src/mbgl/map/map.cpp

```diff
@@ -352,6 +352,8 @@ void Map::Impl::loadStyleJSON(const std::string& json) {
 
     // force style cascade, causing all pending transitions to complete.
     style->cascade(Clock::now(), mode);
+    
+    view.notifyMapChange(MapChangeDidFinishLoadingStyle);
```

Comment:
We probably need a signal for style loading failures. In case of parsing error, the `MapChangeDidFinishLoadingStyle` will be emitted here.